examples/this-that-svc/wasm/this-that-svc.idl

```diff
@@ -20,6 +20,7 @@ type TupleStruct = struct {
 service {
   DoThat : (param: DoThatParam) -> result (struct { str, u32 }, struct { str });
   DoThis : (p1: u32, p2: str, p3: struct { opt str, u8 }, p4: TupleStruct) -> struct { str, u32 };
+  Noop : ();
```

Comment:
Not quite right. The behaviour should be similar to processing a func of the following shape:
```
fn omit_result_type() -> () {}
```
Which will result in:
```
OmitResultType: () -> null;
```
In order to address the issue all is needed is adjusting this code here
https://github.com/gear-tech/sails/blob/a73a73dad9d1fc847a834400500acee04d2e75f8/macros/core/src/shared.rs#L116
Matching against `ReturnType::Default` and returning `()` as Type
https://docs.rs/syn/latest/syn/enum.ReturnType.html#variant.Default

Reply:
All the other code will remain untouched after that.